package-lock.json

```diff
@@ -4338,9 +4235,9 @@
       }
     },
     "node_modules/debug": {
```

Comment:
It looks like you have bumped some relevant dependencies while installing new ones

Reply:
It seems to be relevant, since [socket.io](https://www.npmjs.com/package/socket.io-client?activeTab=dependencies) uses `debug` dependency.

As I see `from package-lock.json` `debug: "^4.3.4",` was used even before this PR. Seems that npm just did some optimisation and removed excess 4.3.1 version